libs/sdk-ui-dashboard/src/plugins/customizationApis/layoutCustomizer.tsx

```diff
@@ -233,4 +254,62 @@ export class DefaultLayoutCustomizer implements IDashboardLayoutCustomizer {
             };
         };
     };
+
+    public getExistingLayoutTransformFn = (): DashboardLayoutExportTransformFn => {
```

Comment:
Should this be public? It's not documented / not defined on interface this class implements.